### ISSUE TITLE
Question : Quel droit faut-il afin de voir les reponses formulaires créées ?

### ISSUE THREAD

**Issue by EC2311** [pluginsGLPI/formcreator#3458]
Bonjour 
Faut-il un droit particulier afin qu'un utilisateur puisse 'visualiser' les reponses créées à l'aide du formulaire :
plugins/formcreator/front/**formanswer.form.php?id**=xxxx , ou xxxx représente alors l'id de la reponse formulaire

je ne souhaite pas mettre le droit de modification entité afin de resoudre ce souci ;) 
ou il faudrait que ce droit s'appuie sur celui de visualisation de ticket ?

Merci

**Comment by EC2311** [comment]
On ne pourrait pas imaginer que ce droit s'appuie sur le droit ticket (voir etc) ? : ce n'est que de la visu et ce serait 'coordonné' avec celui du ticket
car a ce niveau (si on reste ainsi) : on peut voir le ticket et pas la reponse formulaire.

**Comment by btry** [comment]
Non, car il est possible de créer des réponses à des formulaires qui ne génèrent aucune cible. Ou des cibles autres que des tickets.
